portal_multicurrency_totals/demo/res_partner_demo.xml

```diff
@@ -35,10 +32,6 @@
         ]"
         />
     </record>
-    <function
-        model="change.password.wizard"
-        name="change_password_button"
-        eval="[ref('set_password_portal_user')]"
-    />
+    <function model="change.password.wizard" name="change_password_button" eval="[ref('set_password_portal_user')]" />
```

Comment:
Perform autoformatting changes on a separate commit located before this one, as we would do when migratin an OCA module.

Reply:
Done.

Reply:
Please include in the history all commits merged in v14 (#1648).

Reply:
Done.